edx_sga/sga.py

```diff
@@ -851,15 +858,13 @@ def file_storage_path(self, sha1, filename):
         )
         return path
 
-    def is_zip_file_available(self):
+    def is_zip_file_available(self, user):
         """
-        returns True if zip file available.
+        returns True if zip file exists.
         """
-        user = self.get_real_user()
-        require(user)
         zip_file_path = get_zip_file_path(
             user.username,
-            self.block_course_id,
+            unicode(self.block_course_id),
```

Comment:
it is already unicode https://github.com/mitodl/edx-sga/blob/master/edx_sga/sga.py#L170